common.blocks/i-bem-dom/__event-manager/_type/i-bem-dom__event-manager_type_dom.js

```diff
@@ -0,0 +1,218 @@
+/**
+ * @module i-bem-dom__event-manager_type_dom
+ */
+modules.define(
+    'i-bem-dom__event-manager_type_dom',
+    [
+        'i-bem__internal',
+        'inherit',
+        'identify',
+        'objects',
+        'jquery'
+    ],
+    function(
+        provide,
+        BEMINTERNAL,
+        inherit,
+        identify,
+        objects,
+        $) {
+
+var undef,
+    winNode = window,
+    docNode = document,
+    winId = identify(winNode),
+    docId = identify(docNode),
+
+    eventStorage = {},
+
+    buildDomEventManagerParams = function(ctx, defCtx, defSelector, defCls, getEntityCls) {
```

Comment:
Feels like a big part of it (and of buildBemEventManagerParams) can be moved out to some function.